ship_functions.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Aug 15 11:37:18 2019
+
+@author: j-bd
+
+"""
+
+import os
+import argparse
+import shutil
+
+from sklearn.model_selection import train_test_split
+import numpy as np
+import pandas as pd
+import skimage.io
+
+from mrcnn import utils
+from mrcnn import model
+from mrcnn import visualize
+import ship_config
+import ship_dataset
+
+
+def create_parser():
+    '''Get the informations from the operator'''
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-c", "--command", required=True,
```

Comment:
I thing that all those are `required=True` by default. You might not need to specify it

Reply:
It seems it is necessary to specify ```required=True```. I checked with a test. If I don't mention it, the program keeps on going.